### DISCUSSION TITLE
Option to quickly search for Recent Notes

### DISCUSSION THREAD

**Discussion by irmod** [damionx7/Zettel-Notes-Documentation#211] [Feature Requests]
@damionx7, I really like your implementation of Saved Searches. This is just a killer feature!!!

It is similar to Smart Folder from Apple.

Flexible custom filters, thanks to which you can find literally everything in one click.

Tell me, how can I search for recently opened notes? Exactly OPEN, even if I did not change the text of the note, but only viewed the note or copied a fragment from it.

I want to get a list of Recent Notes to quickly switch between recently opened notes.

**Comment by irmod** [comment]
@damionx7, in another application I use this solution to access Recent Notes:

[Recent Files](https://github.com/tgrosinger/recent-files-obsidian)

> This plugin displays a list of most recently opened files in the sidebar. Optionally include paths of files which should be excluded from the list.
>
> That's all there is to it!
> 
> As with the file explorer view, you can:
> 
> - Click items to open, ctrl-click to open in a new pane, right-click for a menu
> - Drag items to an editor to drop a link, to a header to open in a specific pane, or to a file explorer folder to move the file
> - Hover or ctrl-hover to view a content preview (as configured by the "Recent Files" toggle in the "Page Preview" settings)

![Recent_Notes.jpg](https://github.com/user-attachments/assets/0d093dcd-c124-4bf1-b3c8-0ea9069beaed)

Additional settings are useful:

![IMG_20241112_211839.jpg](https://github.com/user-attachments/assets/330ed07d-1be7-4742-a6d4-35b7359cd9ee)

**Comment by damionx7** [comment]
I have added this feature. 
![Screenshot_2024-11-16-20-22-52-69_377fe8767025e4fa6dddb62d6df8f716](https://github.com/user-attachments/assets/0ca462f0-2082-46b3-b2ed-a9799398ada7)

**Reply by damionx7** [reply]
5 recent notes will be shown in left navigation drawer. You can disable this from settings.

**Comment by irmod** [comment]
@damionx7, this is a great solution!!!  I will definitely let you know as soon as I try this feature
